tools/evil-spammer/identity/identityconfig.go

```diff
@@ -0,0 +1,68 @@
+package identity
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+)
+
+type Params map[string]Identities
+type Identities map[string]privKeyURLPair
+type privKeyURLPair [2]string
+
+var Config = Params{}
+
+var identityConfigJSON = `{
```

Comment:
Yes, you could use peer_master or whichever node you will put there, also some random fake id/seed should be ok, but there will be no mana assigned to it with the default snapshot.
Not sure if it would be much of a help to increase mana of the spammer. For sure we should disable activity blocks for spammer node, so we spam only the invalid/incorrect chain. The fork should be observed by the nodes, but without attestations proof it should not work to trigger the chain switch.
The random commitment was supposed to trigger more requests for attestations proofs, as the chain is totally different, but I would need to play with it more, not sure if the weight is the problem.